### PR TITLE
Add loading placeholder for trending project card

### LINKED ISSUE #990: Add placeholder card for loading state for trending projects
## Description

TrendingProjectsCard should have placeholder state while it's loading like ProjectCard.

## Proposal

_A description of the change you'd like to see._

## Screenshots

_If applicable, add screenshots to help explain your problem._

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Closes #990 

## Screenshots or screen recordings

image

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/components/Projects/TrendingProjectCard.tsx [hunk 1]
```diff
@@ -109,26 +111,32 @@ export default function TrendingProjectCard({
...
-            style={{ marginRight: 20, display: 'flex', alignItems: 'center' }}
-          >
-            <div style={rankStyle}>{rank}</div>
+      <div style={cardStyle} className="clickable-border">
+        <div style={{ marginRight: 20, display: 'flex', alignItems: 'center' }}>
+          <div style={rankStyle}>{rank}</div>
+          {metadata ? (
             <ProjectLogo
               uri={metadata.logoUri}
               name={metadata.name}
```
Comment (aeolianeth) [inline src/components/Projects/TrendingProjectCard.tsx:120]: **question**: if we used `?.` here, would we achieve the same outcome as the turnary? 

```suggestion
              uri={metadata?.logoUri}
              name={metadata?.name}
```

File: src/components/Projects/TrendingProjectCard.tsx [hunk 2]
```diff
@@ -140,88 +148,79 @@ export default function TrendingProjectCard({
             >
               {metadata.name}
             </h2>
+          ) : (
+            <Skeleton paragraph={false} title={{ width: 120 }} />
```
Comment (aeolianeth) [inline src/components/Projects/TrendingProjectCard.tsx:152]: **suggestion** we can use the `active` prop to achieve an (arguably) better loading animation: https://ant.design/components/skeleton/#API

**Review by aeolianeth — COMMENTED** [review]
Thanks @jmilldotdev! left some comments, could you also include a video demonstrating the changes? Cheers.

**Comment by jmilldotdev** [comment]
https://user-images.githubusercontent.com/33093632/168320312-3240336b-df7e-473c-aa94-a24f18eb0809.mov

changes made

**Review by aeolianeth — APPROVED** [review]
Nice one thanks for making the changes @jmilldotdev. Approving

**Pull request merged**